index.html

```diff
@@ -0,0 +1,189 @@
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="socials item">
+        <i class="fab fa-brands fa-twitter"></i>
+        <i class="fab fab-brands fa-linkedin-in"></i>
+        <i class="fab fa-brands fa-github"></i>
+        <i class="fa fa-regular fa-hand-peace"></i>
+      </div>
+      
+      <button class="arrow item"><i class="fa fa-solid fa-chevron-down"></i></button>
+      <h2 class="item">My Recent Works</h2>
+      <hr/>
+    </section>
+    <section class="card">
+      <div class="item img-holder">
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder">
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder">
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder">
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder">
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
```

Comment:
- [ ] I would suggest you please use `hx` tag for `Multi-post Stories Gain+Glory` as this is a project title and for better semantic reason

Reply:
completed